CHANGELOG.md

```diff
@@ -34,6 +35,124 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.1.0-rc1
+
+We expect that this release candidate is quite close to the final v1.1.0
+release. However, subsequent breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but we do not recommend
+shipping products based on a release candidate API due to the possibility of
+incompatible changes prior to the final release. The following represents the
+changes since v1.0.0:
+
+## Standard Channel
+
+### GRPCRoute has Graduated to GA 🎉
+
+GRPCRoute has graduated to GA (v1) and is now part of the Standard Channel. If
+you are already using the experimental version GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: @gnossen
+
+### Service Mesh Support has Graduated to GA 🎉
+
+The standard for using Gateway API for Mesh has formally graduated to GA (v1) and is now part of the Standard Channel.
+
+Service mesh support in Gateway API allows service mesh users to use the same API to manage ingress traffic and mesh traffic, reusing the same policy and routing interfaces. In Gateway API v1.1, routes (such as HTTPRoute) can now have a `Service` as a `parentRef`, to control how traffic to specific services behave. For more information, read the [service mesh](https://gateway-api.sigs.k8s.io/mesh/) documentation or the see the list of [integrations](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+
+Leading Contributors: @howardjohn, @keithmattix, @kflynn
+
+### Conformance Profiles and Reports
+
+The Conformance Reports API and the corresponding test suite have been graduated to GA. The Conformance report API has been expanded with the `mode` field (intended to specify the working mode of the implementation), and the `gatewayAPIChannel` (standard or experimental). The `gatewayAPIVersion` and `gatewayAPIChannel` are now filled in automatically by the suite machinery, along with a brief description of the testing outcome.
+The Reports have been reorganized in a more structured way, and the implementations can now add information on how the tests have been run and provide reproduction steps.
+
+Leading Contributors: @mlavacca, @shaneutt
+
+### Port in ParentRefs
```

Comment:
Should this say `Port in ParentRefs (GEP-957) Promoted to GA`, or something like that?

Reply:
`### ParentRef Port field Graduates to GA` ?
`### New GA Field: parentRef.port` ?

There's no way to do this that isn't a bit awkward, but this is going to be a common thing that happens so maybe we should think about how we want to do this in general...